decidim-proposals/lib/decidim/proposals/import/proposal_answer_creator.rb

```diff
@@ -55,6 +56,8 @@ def fetch_resource
 
           proposal.answer = answer
           proposal.answered_at = Time.current
+          @initial_state = proposal.state
```

Comment:
I am not satisfied with this variable `@initial_state` but it is the smallest change, or I can't refactor a bit the method `fetch_resource` which contains logic